src/solver/variable/economy/STStorageInjectionByCluster.h

```diff
@@ -0,0 +1,334 @@
+/*
+** Copyright 2007-2023 RTE
+** Authors: Antares_Simulator Team
+**
+** This file is part of Antares_Simulator.
+**
+** Antares_Simulator is free software: you can redistribute it and/or modify
+** it under the terms of the GNU General Public License as published by
+** the Free Software Foundation, either version 3 of the License, or
+** (at your option) any later version.
+**
+** There are special exceptions to the terms and conditions of the
+** license as they are applied to this software. View the full text of
+** the exceptions in file COPYING.txt in the directory of this software
+** distribution
+**
+** Antares_Simulator is distributed in the hope that it will be useful,
+** but WITHOUT ANY WARRANTY; without even the implied warranty of
+** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+** GNU General Public License for more details.
+**
+** You should have received a copy of the GNU General Public License
+** along with Antares_Simulator. If not, see <http://www.gnu.org/licenses/>.
+**
+** SPDX-License-Identifier: licenceRef-GPL3_WITH_RTE-Exceptions
+*/
+#pragma once
+
+#include "../variable.h"
+
+namespace Antares::Solver::Variable::Economy
+{
+struct VCardSTstorageInjectionByCluster
+{
+    //! Caption
+    static const char* Caption()
+    {
+        return "STS inj by plant";
+    }
+    //! Unit
+    static const char* Unit()
+    {
+        return "P-injection - MWh";
+    }
+
+    //! The short description of the variable
+    static const char* Description()
+    {
+        return "Energy generated by ST storage clusters";
```

Comment:
No energy is generated by ST storages, in fact they lose some if efficiency < 1 !

I'd say something akin to "released".

Reply:
Done. I chose **injected**